docs/user/cli-reference.md

```
@@ -1068,13 +1068,13 @@ bundles/
 └── ...
 ```
 
-**ArgoCD Helm chart structure with `--dynamic`:**
+**Argo CD Helm chart structure with `--dynamic`:**
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)